cmd/mdatagen/templates/readme.connector.md.tmpl

```diff
@@ -0,0 +1,29 @@
+<!-- status autogenerated section -->
+{{- if len .Status.Stability }}
+| Status        |           |
+| ------------- |-----------|
+{{- $stabilityLevels := keys .Status.Stability }}
+| Stability                | [{{ stringsJoin $stabilityLevels "], [" }}] |
```

Comment:
It feels weird to have a stability level here, and another in the table below. I would love to see the two combined somehow

Reply:
I agree but I'm not sure I think this format ends up duplicating information about which signal combinations are supported, and also introduces an awkward concept of `traces_to_metrics`, etc:

| Status        |           |
| ------------- |-----------|
| Stability     | [development]: traces_to_metrics, metrics_to_metrics, logs_to_metrics   |
| Distributions | contrib, sumo |

## Supported Pipeline Types
| [Exporter Pipeline Type] | [Receiver Pipeline Type] |
| ------------------------ | ------------------------ |
| traces | metrics |
| metrics | metrics |
| logs | metrics |


---

I know it diverges from other classes of components but in my opinion it's much clearer to represent this info like this:

| Status        |           |
| ------------- |-----------|
| Distributions | contrib, sumo |

## Supported Pipeline Types
| [Exporter Pipeline Type] | [Receiver Pipeline Type] | Stability |
| ------------------------ | ------------------------ | ---|
| traces | metrics | development |
| metrics | metrics | development |
| logs | metrics | development |

Reply:
I updated the rendering to match this suggestion.